about/faq.rst

```diff
@@ -37,20 +37,20 @@ Which platforms are supported by Godot?
 
 * Windows
 * Mac OS X
-* X11 (Linux, *BSD)
+* X11 (GNU/Linux, \*BSD)
```

Comment:
I don't want to start a political debate, but we're using "Linux" to refer to the operating system everywhere, I don't think we should change now. We can have an issue to discuss it if you want, but that's out of the scope of "small fixes", as we'd have to enforce a *GNU plus Linux* policy everywhere to be consistent.

Reply:
Well, the Asset Library guide (http://docs.godotengine.org/en/latest/tutorials/assetlib/what_is_assetlib.html) is written as GNU/Linux, so I tried my luck.

But I understand, changing it.

Reply:
Hm, that's on me then since I wrote that, unaware of a docs-wide "Linux over GNU/Linux" policy (even though I use Linux over GNU/Linux myself). Might as well change that there's while you're at it.
@YeldhamDev